packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -303,6 +303,8 @@ export enum TransferPanelMainErrorMessage {
   SC_WALLET_ETH_NOT_SUPPORTED
 }
 
+const USDC_L1_ADDRESS = '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48'
```

Comment:
NIT: `USDCE_L1_ADDRESS` for clarity?

Reply:
It's not called USDC.e on L1, only on L2